packages/@ama-sdk/core/src/plugins/core/plugin.ts

```diff
@@ -22,12 +24,25 @@ export interface PluginSyncRunner<T, V> {
   transform(data: V): T;
 }
 
+/**
+ * Interface of a plugin context
+ */
+export interface PluginContext {
```

Comment:
__SUGGESTION__
From what I see the 'Logger' type is used here to export another interface which contains a filed with this type. If I am not mistaken it is the only place where it is used like that. In other places it is used to define directly a type of a property.
If we manage to remove the `logger: Logger` from this interface, we could have the 'o3r/core' as a dev dependency only, because the 'Logger' is not used in the creation of another interface which is exported.
The code will become a bit messy, because where we have the PluginContext we will have PLuginContext & {logger: Logger}
`load(context?: PluginContext & {logger: Logger}):...` but the trade off will be that we will not need `@o3r/core` as a peerDep.

cc: @kpanot @fpaul-1A